examples/multimodal/wds.py

```diff
@@ -9,25 +7,36 @@
     .filter(C("file.name").glob("00000000.tar"))
     .settings(cache=True)
     .gen(laion=process_webdataset(spec=WDSLaion), params="file")
+    .save()  # materialize chain to avoid downloading data multiple times
```

Comment:
For this example, the run time before this change was > 2hrs (for me). It now takes ~25 minutes to run.